src/templates/package.json

```diff
@@ -23,7 +23,15 @@
 		"@graphql-mesh/transform-resolvers-composition": "^0.12.111",
 		"@graphql-mesh/transform-type-merging": "^0.4.56",
 		"@graphql-mesh/types": "^0.87.1",
-		"graphql": "^16.6.0"
```

Comment:
Wondering why the name is left empty? On[ Line 2](https://github.com/adobe/aio-cli-plugin-api-mesh/pull/85/files#diff-7ef35b6e03f05ade7e4b295e6443d06a397918c9fd1f55e096e0ef56f3d25b39R2)

Reply:
Yeah its a template. Will be filled out during the init command usage.